patches/minecraft/net/minecraft/entity/item/EntityMinecart.java.patch

```diff
@@ -158,16 +169,39 @@
                  ++this.field_70163_u;
          }
  
-@@ -509,7 +541,7 @@
-             }
-         }
+@@ -490,26 +519,12 @@
+         this.field_70179_y = d5 * d2 / d3;
+         Entity entity = this.func_184188_bt().isEmpty() ? null : (Entity)this.func_184188_bt().get(0);
  
+-        if (entity instanceof EntityLivingBase)
```

Comment:
Should this not rather be dead-coded?

Reply:
What? This is moved to a block below. Modded minecarts can change it in whatever way they want.